<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1216</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="my_label" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mouse_movement">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>    Mouse Movement</string>
    </property>
   </widget>
   <widget class="QLabel" name="mouse_pressed">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>110</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>80</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mouse Pressed</string>
    </property>
   </widget>
   <widget class="Line" name="x_axis">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>225</y>
      <width>700</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 85, 255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="y_axis">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>0</y>
      <width>1</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_axes">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>40</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Axes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Draw">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="circle_radius">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>60</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="draw_circle">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Circle</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="draw_line">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_point1">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>0</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set point 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_point2">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>30</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set point 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>190</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Grid</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>190</y>
      <width>44</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="set_p1">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>240</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set P1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_p2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>270</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set P2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="draw_DDA">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>120</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw DDA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="draw_Bresenham">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>160</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Bresenham</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="axes">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>150</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Axes</string>
    </property>
   </widget>
   <widget class="QLabel" name="DDA_Time">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Bresenham_Time">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="radius">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>250</y>
      <width>44</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="draw_Polar">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>290</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Polar</string>
    </property>
   </widget>
   <widget class="QLabel" name="Polar_Time">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>290</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>250</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Radius</string>
    </property>
   </widget>
   <widget class="QPushButton" name="draw_Bresenhsam_mid_point">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>330</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Bresenham mid-point</string>
    </property>
   </widget>
   <widget class="QLabel" name="Bresenham_mid_point_Time">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>330</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>210</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Circle:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>90</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Line:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>400</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ellipse:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>430</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ry</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>430</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rx</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="rx">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>430</y>
      <width>44</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="ry">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>430</y>
      <width>44</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="draw_Polar_Ellipse">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>480</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Polar</string>
    </property>
   </widget>
   <widget class="QLabel" name="Polar_Time_Ellipse">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>480</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Bresenham_mid_point_Time_Ellipse">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>520</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="draw_Bresenhsam_mid_point_Ellipse">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>520</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Bresenham mid-point</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boundary_fill_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>400</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Boundary Fill 4 connected</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>360</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Filling:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1216</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>my_label</class>
   <extends>QLabel</extends>
   <header location="global">my_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
